src/lib/config.ts

```diff
@@ -1254,6 +1241,20 @@ export async function getAsync(
     }
 }
 
+/*
+ * Replaces the sync storage with your current userconfig. Does not merge: it overwrites.
```

Comment:
Should rename usercinfig to configuration. Explain more explicitly that this is for synchronisation a bit like git.